### DISCUSSION TITLE
Skin Editor: Option to always display a component.

### DISCUSSION THREAD

**Discussion by Bryceed** [ppy/osu#23236] [Ideas] [answered]
In terms of customization, it would be nice if users could personalize their experience by pinning a component to be always visible, even on the ‘focus mode’, such as an error meter or accuracy display.

**Comment by Theighlin** [comment]
This is planned in the skin editor.

**Reply by Bryceed** [reply]
Any evidence? (to mark as "answered")

**Reply by Joehuu** [reply, marked as answer]
There is already an issue opened, see https://github.com/ppy/osu/issues/12039#issuecomment-856453309.
